pkg/kubelet/kubelet.go

```diff
@@ -2074,7 +2073,7 @@ func (kl *Kubelet) canAdmitPod(pods []*api.Pod, pod *api.Pod) (bool, string, str
 func (kl *Kubelet) syncLoop(updates <-chan kubetypes.PodUpdate, handler SyncHandler) {
 	glog.Info("Starting kubelet main sync loop.")
 	kl.resyncTicker = time.NewTicker(kl.resyncInterval)
```

Comment:
(optional) while your at it, do you want to modify resyncTicker be a function parameter like housekeepingTicker, and remove it from the kubelet struct?